mcstatus/pinger.py

```diff
@@ -3,53 +3,52 @@
 import datetime
 import json
 import random
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, TypedDict
```

Comment:
We must use `typing_extensions.TypedDict`, see [PEP-655 Usage in Python <3.11](https://peps.python.org/pep-0655/#usage-in-python-3-11).

Reply:
I wonder why the CI checks (which include pyright) passed though, I'd expect that since we're supposed to use the `typing_extensions` version with these, pyright would complain.

Reply:
Entire change to this file is removed. Looks like it can't be changed until python 3.11 years from now.

Reply:
Could we at least use `TypedDict = dict` in `else` statement, under `TYPE_CHECKING` block? Would be better, that adding everything in the block.

Reply:
> I wonder why the CI checks (which include pyright) passed though, I'd expect that since we're supposed to use the `typing_extensions` version with these, pyright would complain.

Because we expect that this can not be in dict, and do not try to get it with `.`. [Look here](https://github.com/py-mine/mcstatus/blob/94bd9efc3d0d26421c37076d62bcc591325d254b/mcstatus/pinger.py#L291).

Reply:
Seems misleading to have a TypedDict not really be a TypedDict.

Reply:
Only in runtime. It is actually working the same, but instead of replacing `TypedDict` with dict, we replace all our typed dicts.
https://github.com/py-mine/mcstatus/blob/94bd9efc3d0d26421c37076d62bcc591325d254b/mcstatus/pinger.py#L47-L52

Reply:
Would you concur that `TypedDict` and `dict` are not the same thing?
Given that they are not, why would we be compelled falsely claim that they are by writing `TypedDict = dict`?